redis/src/aio/mod.rs

```diff
@@ -97,7 +97,7 @@ async fn execute_connection_pipeline(
     rv: &mut impl ConnectionLike,
     (pipeline, instructions): (crate::Pipeline, ConnectionSetupComponents),
 ) -> RedisResult<AuthResult> {
-    if pipeline.len() == 0 {
```

Comment:
These changes are a little annoying, and the result of a rabbithole clippy was leading me down :/.

After creating the method `pipeline.is_empty()` Clippy raises the below error on both `mod.rs`, and `connection.rs`.

```
error: length comparison to zero
   --> redis/src/aio/mod.rs:100:8
    |
100 |     if pipeline.len() == 0 {
    |        ^^^^^^^^^^^^^^^^^^^ help: using `is_empty` is clearer and more explicit: `pipeline.is_empty()`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#len_zero
    = note: `-D clippy::len-zero` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::len_zero)]`
```